CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ common_find_package(BBPTestData)
 common_find_package(Boost REQUIRED COMPONENTS unit_test_framework
                                               program_options system)
 common_find_package(Brion REQUIRED)
-common_find_package(CUDA 4.0)
+#common_find_package(CUDA 4.0)
```

Comment:
Write a comment here and in the commit message about why this is commented out.
At the same time, can't you uncomment this and prefer the CPU implementation in code?